persistent-qq/src/Database/Persist/Sql/Raw/QQ.hs

```diff
@@ -121,56 +122,27 @@ timesCommaSeparated n = intercalate "," . replicate n
 
 makeExpr :: TH.ExpQ -> [Token] -> TH.ExpQ
 makeExpr fun toks = do
-    TH.infixE
-        (Just [| uncurry $(fun) . second concat |])
-        ([| (=<<) |])
-        (Just $ go toks)
```

Comment:
I rewrote the TH in this function to be a bit more concise and use more quotes vs manually splicing TH functions together.